README.md

```diff
@@ -1,27 +1,151 @@
 ###Ruby Wrapper to the PlotLy REST API
 
-**Example Usage**
+**Connecting to the API**
 
 ```ruby
 
 plotly = PlotLy.new('username', 'api key')
 
-args = [[0,1,2], [3,1,6]]
-
-kwargs = {
-  filename: 'plot from API',
-  fileopt: 'overwrite',
-  style: { type: 'bar' },
-  traces: [0,3,5],
-  layout: {
-    title: 'Experimental Data'
-  },
-  world_readable: true
-}
-
-plotly.plot(args, kwargs) do |response|
-  puts response['url']
-end
+```
+
+**Time Series**
+
+<div>
+    <a href="https://plot.ly/~louism/7/" target="_blank" title="Ruby API Time Series Demo" style="display: block; text-align: center;"><img src="https://plot.ly/~louism/7.png" alt="Ruby API Time Series Demo" style="max-width: 100%;"  onerror="this.onerror=null;this.src='https://plot.ly/404.png';" /></a>
+    <script data-plotly="louism:7"  src="https://plot.ly/embed.js" async></script>
+</div>
+
+
+```ruby
+
+    data = { 
+      x: ['2013-10-04 22:23:00', '2013-11-04 22:23:00', '2013-12-04 22:23:00'],
+      y: [1, 3, 6]
+    }
+
+    args = {
+      filename: 'ruby_test_time_series',
+      fileopt: 'overwrite',
+      style: { type: 'scatter' },
+      layout: {
+        title: 'Ruby API Time Series Demo'
+      },
+      world_readable: true
+    }
+
+    plotly.plot(data, args) do |response|
+      puts response['url']
+    end
+    
+```
+
+
+**Bar Plot**
+
+<div>
+    <a href="https://plot.ly/~louism/3/" target="_blank" title="Ruby API Bar Plot Demo" style="display: block; text-align: center;"><img src="https://plot.ly/~louism/3.png" alt="Ruby API Bar Plot Demo" style="max-width: 100%;"  onerror="this.onerror=null;this.src='https://plot.ly/404.png';" /></a>
+    <script data-plotly="louism:3"  src="https://plot.ly/embed.js" async></script>
+</div>
+
+```ruby
+
+    data = [
+      { x: ['giraffes', 'orangutans', 'monkeys'],
+        y: [20, 14, 23]
+      }, 
+      { x: ['giraffes', 'orangutans', 'monkeys'],
+        y: [12, 18, 29]
+      }
+    ]
+    
+    args = {
+      filename: 'ruby_test_bar_plot',
+      fileopt: 'overwrite',
+      style: { type: 'bar', barmode: 'group' },
+      layout: {
+        title: 'Ruby API Bar Plot Demo'
+      },
+      world_readable: true
+    }
+
+    plotly.plot(data, args) do |response|
+      puts response['url']
+    end    
+```
+
+**Bar Plot with Time Markers**
```

Comment:
Looks great, but please make years MySQL string format (ie - "2013-10-04" ...). Otherwise Plotly thinks the years are floats and you get this issue zooming in:
![image](https://cloud.githubusercontent.com/assets/1865834/8896574/9fa3f466-33ce-11e5-82e0-9b06646ce950.png)